test/distributed/pipelining/schedule_registry.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# Owner(s): ["oncall: distributed"]
+# This file is a Schedule zoo for testing torch.distributed.pipelining.
+# It includes schedules designed purely for testing purposes
+from typing import Any, Callable, Dict, List, Optional
+
+import torch
+from torch.distributed.pipelining.schedules import _ComputationType, _Action, PipelineScheduleMulti
+from torch.distributed.pipelining.stage import _PipelineStageBase
+
+F = _ComputationType.FORWARD
+B = _ComputationType.BACKWARD
+
+class ScheduleUnevenLooped(PipelineScheduleMulti):
```

Comment:
nit: it's not really looped, its V, isn't it?

I like the idea of the schedule registry

Reply:
Oops yeah, will update the name!